backend/apps/owasp/api/search/project.py

```diff
@@ -36,3 +36,7 @@ def get_projects(query, attributes=None, limit=25):
 def projects(request):
     """Search projects API endpoint."""
     return JsonResponse(get_projects(request.GET.get("q", "")), safe=False)
+
+def active_projects_count(request):
```

Comment:
Great idea. Do you think we could incorporate this data into the existing endpoint for projects? Basically, return a dict -- I think we can keep the API endpoints complexity lower this way.
```python
{"active_projects": Project.active_projects_count(), 
 "items": get_projects(request.GET.get("q", "")), safe=False
}```

Reply:
Totally agree with this! Thanks @arkid15r !
Have pushed a commit :)